packages/legend-application-query/src/stores/data-space/DataSpaceQueryCreatorStore.ts

```diff
@@ -342,4 +348,29 @@ export class DataSpaceQueryCreatorStore extends QueryEditorStore {
       ),
     );
   }
+
```

Comment:
what happens when user enters the default route of query (no project info). 
Do we have a search specification for this case ?

Reply:
by default,
the function is empty 

https://github.com/finos/legend-studio/pull/3212/files#diff-5dae4710ada9d402191df7256d5cc7d4502e3b40a8f1b9acf960143f7ee6adfeR437

Reply:
yeah but what does that return in terms of queries ?